lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -15,7 +15,7 @@ include_once 'dosomething_user.theme.inc';
  */
 function dosomething_user_preprocess_page(&$vars) {
   $school_api_endpoint = variable_get('dosomething_user_school_api_endpoint', 'http://lofischools.herokuapp.com/search');
-  $date_format = (dosomething_settings_get_geo_country_code() === 'US') ? 'MM/DD/YYYY' : 'DD/MM/YYYY';
```

Comment:
Yeah, I think I was confused since you pointed me to that file location @sbsmith86. So that actually deals with setting how the data validates, but not how the placeholder, etc is presented in the field. Gonna change it back, cause that's not what we want for that location. Thanks for the catch @sbsmith86 & @DFurnes